blockchain/.env.example

```diff
@@ -20,3 +20,6 @@ SPROUT_VIEWKEY=""
 
 # extended public seed
 BIP32_XPUB=""
+
+# Sentry URL
+SENTRY_DSN="https://2f7205bc6f7d4ef1983977089c34f631@sentry.io/1374764"
```

Comment:
Should the DSN be checked in here?

Reply:
WHOOPS.

Reply:
Removed, and deleted this DSN key since this is now public.